example/example_generated_test.go

```diff
@@ -0,0 +1,1447 @@
+	assert.NotNil(t, result)
+}
+
+func TestPayoutList1(t *testing.T) {
+	params := &stripe.PayoutListParams{}
+	params.Filters.AddFilter("limit", "", "3")
+	result := payout.List(params)
+	assert.NotNil(t, result)
+}
+
+func TestPayoutCancel1(t *testing.T) {
+	result, _ := payout.Cancel("po_xxxxxxxxxxxxx", nil)
+	assert.NotNil(t, result)
+}
+
+func TestPayoutReverse1(t *testing.T) {
+	result, _ := payout.Reverse("po_xxxxxxxxxxxxx", nil)
+	assert.NotNil(t, result)
+}
+
+func TestProductCreate1(t *testing.T) {
+	params := &stripe.ProductParams{Name: stripe.String("Gold Special")}
+	result, _ := product.New(params)
+	assert.NotNil(t, result)
+}
+
+func TestProductRetrieve1(t *testing.T) {
+	result, _ := product.Get("prod_xxxxxxxxxxxxx", nil)
+	assert.NotNil(t, result)
+}
+
+func TestProductUpdate1(t *testing.T) {
+	params := &stripe.ProductParams{}
+	params.AddMetadata("order_id", "6735")
+	result, _ := product.Update("prod_xxxxxxxxxxxxx", params)
+	assert.NotNil(t, result)
+}
+
+func TestProductList1(t *testing.T) {
+	params := &stripe.ProductListParams{}
+	params.Filters.AddFilter("limit", "", "3")
+	result := product.List(params)
+	assert.NotNil(t, result)
+}
+
+func TestProductDelete1(t *testing.T) {
+	result, _ := product.Del("prod_xxxxxxxxxxxxx", nil)
+	assert.NotNil(t, result)
+}
+
+func TestPriceCreate1(t *testing.T) {
+	params := &stripe.PriceParams{
+		UnitAmount: stripe.Int64(2000),
+		Currency:   stripe.String(string(stripe.CurrencyUSD)),
+		Recurring:  &stripe.PriceRecurringParams{Interval: stripe.String("month")},
+		Product:    stripe.String("prod_xxxxxxxxxxxxx"),
+	}
+	result, _ := price.New(params)
+	assert.NotNil(t, result)
+}
+
+func TestPriceRetrieve1(t *testing.T) {
+	result, _ := price.Get("price_xxxxxxxxxxxxx", nil)
+	assert.NotNil(t, result)
+}
+
+func TestPriceUpdate1(t *testing.T) {
+	params := &stripe.PriceParams{}
+	params.AddMetadata("order_id", "6735")
+	result, _ := price.Update("price_xxxxxxxxxxxxx", params)
+	assert.NotNil(t, result)
+}
+
+func TestPriceList1(t *testing.T) {
+	params := &stripe.PriceListParams{}
+	params.Filters.AddFilter("limit", "", "3")
+	result := price.List(params)
+	assert.NotNil(t, result)
+}
+
+func TestRefundCreate1(t *testing.T) {
+	params := &stripe.RefundParams{Charge: stripe.String("ch_xxxxxxxxxxxxx")}
+	result, _ := refund.New(params)
+	assert.NotNil(t, result)
+}
+
+func TestRefundRetrieve1(t *testing.T) {
+	result, _ := refund.Get("re_xxxxxxxxxxxxx", nil)
+	assert.NotNil(t, result)
+}
+
+func TestRefundUpdate1(t *testing.T) {
+	params := &stripe.RefundParams{}
+	params.AddMetadata("order_id", "6735")
+	result, _ := refund.Update("re_xxxxxxxxxxxxx", params)
+	assert.NotNil(t, result)
+}
+
+func TestRefundList1(t *testing.T) {
+	params := &stripe.RefundListParams{}
+	params.Filters.AddFilter("limit", "", "3")
+	result := refund.List(params)
+	assert.NotNil(t, result)
+}
+
+// TODO Token test cases skipped because not all Token methods are supported by GO
```

Comment:
Could we deduplicate these comments, or maybe remove them altogether? As long as we have todo items inside the codegen source code, that is enough. We don't need to track the TODOs here as well.

Reply:
wil do